src/ansys/fluent/core/session.py

```diff
@@ -266,6 +275,40 @@ def force_exit_container(self) -> None:
         """Terminate Docker container session."""
         self.fluent_connection.force_exit_container()
 
+    def upload(self, file_name: str, remote_file_name: Optional[str] = None):
+        """Upload a file on the server supported by `PyPIM<https://pypim.docs.pyansys.com/version/stable/>`.
+
+        Parameters
+        ----------
+        file_name : str
+            file name
+        remote_file_name : str, optional
+            remote file name, by default None
+        """
+        uploader = _get_uploader(self.fluent_connection)
```

Comment:
Creating a new `_get_uploader` method seems a little excessive, as (if I'm understanding right) this could be replaced with just:
```suggestion
        uploader = PimFileTransferService(self.fluent_connection._remote_instance)
```

Reply:
Yes, we can update this. I just followed our discussion.

Reply:
It's removed now.